tests/Doctrine/Tests/DBAL/Functional/StatementTest.php

```diff
@@ -320,4 +323,65 @@ public function testFetchInColumnMode(): void
 
         self::assertEquals(1, $result);
     }
+
+    public function testFetchAllWithOneArgumentOnExternalPDO(): Connection
+    {
+        $driver = $this->connection->getDriver();
+
+        if (! $driver instanceof PDOSqliteDriver) {
+            $this->markTestSkipped('External PDO connection tests run only with an SQLite configuration.');
```

Comment:
Why does this test require SQLite?

Reply:
As I mentioned in https://github.com/doctrine/dbal/pull/4173#issuecomment-697987861, I think it's overkill to test every PDO driver here, so I've set it up for in-memory SQLite only. And I thought it's more logical to only actually run this test when we're running with an SQLite config, then?

Actually, I've had to remove this check as I'm not extending `DbalFunctionalTestCase` anymore, so I don't have access to the currently configured driver. May I'd need to re-introduce a condition before running these tests, either reading from the global config, or at least check that PDO SQLite is installed?

Reply:
> I think it's overkill to test every PDO driver here […]

Overkill in what? Run time, development time, maintenance? The test suite is designed to run all functional tests with all supported configurations. You don't have to reimplement that from scratch.